index.html

```diff
@@ -13963,41 +13991,50 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
             </caption>
             <thead>
               <tr>
-                <th scope="col">Value</th>
+                <th scope="col">Value (keyword)</th>
+                <th scope="col">State</th>
                 <th scope="col">Description</th>
               </tr>
             </thead>
             <tbody>
               <tr>
-                <th class="value-name" scope="row"><strong class="default">false (default)</strong></th>
+                <th class="value-name" scope="row"><strong class="default">false</strong></th>
+                <th class="value-name" scope="row">False</th>
                 <td class="value-description">Indicates the element does not have a popup.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">true</th>
+                <th class="value-name" scope="row">True</th>
                 <td class="value-description">Indicates the popup is a <rref>menu</rref>.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">menu</th>
+                <th class="value-name" scope="row">Menu</th>
                 <td class="value-description">Indicates the popup is a <rref>menu</rref>.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">listbox</th>
+                <th class="value-name" scope="row">Listbox</th>
                 <td class="value-description">Indicates the popup is a <rref>listbox</rref>.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">tree</th>
+                <th class="value-name" scope="row">Tree</th>
                 <td class="value-description">Indicates the popup is a <rref>tree</rref>.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">grid</th>
+                <th class="value-name" scope="row">Grid</th>
                 <td class="value-description">Indicates the popup is a <rref>grid</rref>.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">dialog</th>
+                <th class="value-name" scope="row">Dialog</th>
                 <td class="value-description">Indicates the popup is a <rref>dialog</rref>.</td>
               </tr>
             </tbody>
           </table>
+          <p>The attribute's <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a> is the Undefined state, and its <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> is the Menu state.</p>
```

Comment:
Migrated comment from @keithamus ([original](https://github.com/w3c/aria/pull/2413/files#r1983930085)):

Again, here, the missing value default being the `Undefined` state means `.ariaHasPopup` will return `"undefined"` which I don't think is desirable. Instead:

```suggestion
			<p>The attribute has no <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a>, and its <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> is the Menu state.</p>
```

Reply:
@keithamus Thanks! `aria-haspopup` doesn't actually support an undefined state so I've removed this. The updated enumerated attribute for aria-haspopup:

Keyword → State Mapping:
- false, (empty string) → False
- true, menu → Menu
- listbox → Listbox
- tree → Tree
- grid → Grid
- dialog → Dialog

Missing Value Default:
- (null)

Invalid Value Default:
- false